app/src/main/java/com/example/appmarvel/activity/CharactersListActivity.kt

```diff
@@ -34,7 +37,9 @@ class CharactersListActivity : AppCompatActivity(), KoinComponent {
     }
 
     private fun showCharactersListOnTextView(charactersList: List<Character>) {
-        binding.textViewCharactersActivityCharactersList.text = charactersList.toString()
+        characterAdapter = CharacterAdapter(charactersList)
```

Comment:
as you are not using this variable in other place I think that you do not need it, create the adapter in line 42 directly

Reply:
Done